### PR TITLE
Implement function to get details on a specific menu item

### LINKED ISSUE #27: Get menu item details
- [X] Get menu item route
- [X] Get menu item model

### PR DESCRIPTION
**Pull request by WinsonChen114**

### CODE REVIEW

File: models/menu.js [hunk 1]
```diff
@@ -70,6 +70,25 @@ class Menu {
         console.log("result.rows.length: ", result.rows.length)
```
Comment (LifeBringer) [inline models/menu.js]: Delete/comment out

**Review by LifeBringer — APPROVED** [review]
See comment

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Delete/comment out

File: models/menu.js (hunk 1 above)
Comment (WinsonChen114) [reply models/menu.js]: Done

**Pull request merged**